docs/workflow_solutions/shell.md

```diff
@@ -608,8 +608,56 @@ To change a directory and all of its contents recursively use `chgrp -hR <new-gr
 
 ### Manage researcher access to files and directories (`getfacl`, `setfacl`)
 
-<!-- markdownlint-disable MD046 -->
-!!! construction
+Access Control Lists (ACLs) provide the ability to set permissions for individual users or groups on specific files and directories.
 
-    Under construction.
-<!-- markdownlint-enable MD046 -->
+**Key ACl commands**:
+
+- `getfacl`: Command to get the ACL of a file or directory.
+- `setfacl`: Command to set or modify the ACL of a file or directory.
+
+#### Viewing Current ACLs (`getfacl`)
+
+The `getfacl` command is used to view the current ACLs set on a file or directory. When you run this command on a file, for example `test.txt`, the command `getfacl test.txt` will display detailed information about the ACLs, including who has access to the file and what permissions they have.
+
+```bash
+# file: test.txt
+# owner: bhbelay
+# group: rc-datasci
+user::rw-
+group::rw-
+other::r--
+```
+
+The output shows that for the file `test.txt`, the owner is `bhbelay` and the group associated with the file is `rc-datasci`. The other lines of the output:
```

Comment:
Compare owner and group with permissions. This will help people who are already familiar with permissions more readily understand ACLs.